La branche dev est cassée

Nous avons la branche dev à nouveau cassée. Tous les tests passent mais, après ça, l'erreur suivante est afficher dans Travis :

``` console
back_mysql runtests: commands[1] | flake8
./zds/utils/templatetags/emarkdown.py:33:37: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:33:39: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:35:34: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:35:36: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:37:45: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:37:47: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:42:38: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:42:40: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:45:41: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:45:43: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:48:42: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:48:44: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:50:35: E251 unexpected spaces around keyword / parameter equals
./zds/utils/templatetags/emarkdown.py:50:37: E251 unexpected spaces around keyword / parameter equals
ERROR: InvocationError: '/home/travis/build/zestedesavoir/zds-site/.tox/back_mysql/bin/flake8'
___________________________________ summary ____________________________________
ERROR:   back_mysql: commands failed
The command "tox $TEST_APP" exited with 1.
```

Quelqu'un comprend le problème ?

--- a/zds/utils/templatetags/emarkdown.py
+++ b/zds/utils/templatetags/emarkdown.py
@@ -30,24 +30,24 @@ def get_markdown_instance(inline=False, js_support=False):
     zdsext = ZdsExtension({"inline": inline, "emoticons": smileys, "js_support": js_support})
     # Generate parser
     md = markdown.Markdown(extensions=(zdsext,),
-                           safe_mode = 'escape',
+                           safe_mode='escape',
                            # Protect use of html by escape it
-                           inline = inline,
+                           inline=inline,
                            # Parse only inline content.
-                           enable_attributes = False,
+                           enable_attributes=False,
                            # Disable the conversion of attributes.
                            # This could potentially allow an
                            # untrusted user to inject JavaScript
                            # into documents.
-                           tab_length = 4,
+                           tab_length=4,
                            # Length of tabs in the source.
                            # This is the default value
-                           output_format = 'html5',
+                           output_format='html5',
                            # html5 output
                            # This is the default value
-                           smart_emphasis = True,
+                           smart_emphasis=True,
                            # Enable smart emphasis for underscore syntax
-                           lazy_ol = True,
+                           lazy_ol=True,
                            # Enable smart ordered list start support
                            )
     return md

